bin/index.js

```diff
@@ -39,6 +39,9 @@ function gulpTerser(options) {
       res.error.filePath = file.path
       res.error.fileContent = (typeof build === 'string') ? build : build[res.error.filename]
 
+      // use system new line instead terser new line
+      res.error.fileContent = res.error.fileContent.replace(/\n\r\u2028\u2029/g, os.EOL)
```

Comment:
I think that should be e.g.: `"gulp-terser-js": "https://github.com/A-312/gulp-terser-js.git#fix-11"`

That regex only matches those 4 characters in a row in that order and so makes no difference in the cases I tested.

This works in my tests:
`res.error.fileContent = res.error.fileContent.replace(/\r\n|\r|\n|\u2028|\u2029/g, os.EOL)`
The `|` means "or" so it matches any of the 4 characters individually, but it accounts for `\r\n` being counted as a single return. What I don't know 100% is if some regex find-and-replace libs would get into an infinite loop if they replace e.g. a `\n` with a `\r\n` and don't advance past the `\n` they insert, but this did not happen when I tested.

I tested 1 file with `\n` line endings on a Linux machine and a Windows machine, and the same file with the line endings changed to `\r\n` on both machines. In all 4 cases, the results were unchanged by this PR (`TypeError` on Windows with `\n`), but working with the LoC I submitted above (showed the JS compilation error text as expected with the correct line number); gulp-terser-js did not throw an unexpected error itself.)